haskell/stack.yaml

```diff
@@ -30,7 +30,9 @@ extra-deps:
     https://gitlab.com/morley-framework/morley-metadata.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    abc385928e69b6429595c6169b2494165aae9c77 # master
+    #7b3ee35067f368e627029d71f3b5453b430bac0d # master
+    7dc27b097976481c3dca171a1b86d8cdb3e9a0a5 # sras/add-check-view
```

Comment:
Overall LGTM, I haven't checked out the MR for this branch yet, but I will tomorrow.

Hopefully we can merge this and the `morley-ledgers` one soon